vmm/src/vm_config.rs

```diff
@@ -68,6 +68,7 @@ pub struct CpusConfig {
     pub affinity: Option<Vec<CpuAffinity>>,
     #[serde(default)]
     pub features: CpuFeatures,
+    pub nested: Option<bool>,
```

Comment:
Does this make sense to be an `Option` type - and rather a bool with `#[serde(default = true)]`

Reply:
This seems reasonable to me as  we default the same value for both KVM and MSHV. option is unnecessary.